content/docs/reference/service-urls.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Service URLs
+id: service-urls
+description: This page covers all of Pomerium's service URL settings, including authenticate and authorize external and internal service URLs.
+keywords:
+  - authenticate service url
+  - authenticate internal service url
+  - authenticate_service_url
+  - authenticate_internal_service_url
+  - authorize service url
+  - authorize_service_url
+  - authorize internal service url
+  - authorize_internal_service_url
+pagination_prev: null
+pagination_next: null
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import PomeriumAuthenticateServices from '@site/content/docs/admonitions/\_pomerium-authenticate-services.mdx';
+
+# Service URL Settings
+
+## Authenticate Service URL
+
+The **Authenticate Service URL** setting defines the externally accessible URL where Pomerium redirects end users (clients) to authenticate against an identity provider.
+
+If Pomerium is running in [split service mode](/docs/internals/configuration#all-in-one-vs-split-service-mode), each individual Pomerium service (except the Databroker service) requires the authenticate service URL in its configuration. These services include the Proxy, Authenticate, and Authorize services.
```

Comment:
It actually is needed by the Databroker. The Databroker doesn't make a connection to it, but it ends up disabling session refresh because it thinks we're using hosted authenticate if undefined.